content/english/faq.md

```diff
@@ -60,6 +60,36 @@ Starting with HedgeDoc 2.0, mermaid is used for rendering sequence-diagrams. The
     ```
 {{< /faq-entry >}}
 
+{{< faq-entry title="Why did the way tags are set change?" draft=true >}}
```

Comment:
I would split this into two questions because that are two different changes.

Reply:
done